lib/rules/indent.js

```diff
@@ -61,6 +66,24 @@ module.exports = function(context) {
         }
     }
 
+    if (typeof options.VariableDeclarator === "number") {
+        options.VariableDeclarator = {
+            var: options.VariableDeclarator,
+            let: options.VariableDeclarator,
+            const: options.VariableDeclarator
+        };
+    } else {
```

Comment:
Can't use `object-assign` since it overrides everything. What I need here is `_.defaults`: http://underscorejs.org/#defaults